json/src/test/scala/spray/json/CollectionFormatsSpec.scala

```diff
@@ -20,7 +18,9 @@ class CollectionFormatsSpec extends WordSpec with DefaultJsonProtocol {
     }
   }
 
+  /*
```

Comment:
Commented tests?

Reply:
Was hoping to get it working but dropping array support seems best.